app-chores/package.json

```diff
@@ -0,0 +1,14 @@
+
+{
+  "name": "app-chores",
+  "version": "0.0.1",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+	"start": "node app.js",
+    "test": "test.js"
```

Comment:
Test no forma parte de este milestone. Tenéis que hacer un producto mínimo

Reply:
Me pongo con ello :ok_hand: